dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx

```diff
@@ -200,14 +200,30 @@ const columnStyle = css`
 	}
 `;
 
+const columnStyleWithPageSkin = css`
+	${columnStyle};
+	${from.leftCol}, ${from.wide} {
```

Comment:
This is redundant, you only need the lowest of the two breakpoints, as `wide` is larger than `leftCol`.

Reply:
Done aa0367810f0f3dada125612cb1c53b228f3bd821